src/testing/internal/schemas/friendList.graphql17Alpha2.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  GraphQLID,
+  GraphQLInt,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLSchema,
+  GraphQLString,
+} from "graphql-17-alpha2";
+
+const friendType = new GraphQLObjectType({
+  fields: {
+    id: { type: GraphQLID },
+    name: { type: GraphQLString },
+    nonNullName: { type: new GraphQLNonNull(GraphQLString) },
+  },
+  name: "Friend",
+});
+
+const query = new GraphQLObjectType({
+  fields: {
+    scalarList: {
+      type: new GraphQLList(GraphQLString),
```

Comment:
Adding a resolver like this would allow you to slowly stream in lists for tests without having to resort to `enqueueSubsequentChunk`.
```suggestion
      async *resolve(rootValue) {
        for (const value of rootValue.scalarList || []) {
          await wait(3);
          yield value;
        }
      },
```
or even make it possible to pass that in via variable:
```suggestion
      async *resolve(
        rootValue,
        { chunks } = { chunks: [rootValue.scalarList] }
      ) {
        for (const value of chunks) {
          await wait(5);
          yield value;
        }
      },
```